lib/sprite_animation.dart

```diff
@@ -60,65 +94,30 @@ class SpriteAnimation {
     frames = sprites.map((s) => SpriteAnimationFrame(s, stepTime)).toList();
   }
 
-  /// Automatically creates a sequenced animation, that is, an animation based on a sprite sheet.
```

Comment:
Maybe this comment describing the arguments should be on the SpriteAnimationOpts.sequenced now?

Reply:
Yep. I have deleted it because it was quite outdated, when I do the docs, I will try to update and create new ones on `SpriteAnimationOpts`